src/agent.rs

```diff
@@ -398,9 +399,9 @@ struct RemoteAgent<AGN: Agent> {
 }
 
 impl<AGN: Agent> RemoteAgent<AGN> {
-    pub fn new(worker: &Value, slab: Shared<Slab<Callback<AGN::Output>>>) -> Self {
+    pub fn new(worker: Value, slab: Shared<Slab<Callback<AGN::Output>>>) -> Self {
```

Comment:
This is a breaking change, is it necessary?

Reply:
Hmm.. is it? Its just clear for me that reference is not needed there.
But if its a design...

Reply:
You're right, it's ok because `RemoteAgent` is not public